configure.py

```diff
@@ -1235,8 +1236,14 @@ def maybe_encode_env(env):
     if environ_cp.get('TF_NCCL_VERSION', None):
       cuda_libraries.append('nccl')
 
+  find_cuda_path = 'third_party/gpus/find_cuda_config.py'
+  if not Path(find_cuda_path).is_file():
+      find_cuda_path = Path().glob('**/third_party/gpus/find_cuda_config.py')
```

Comment:
Please use `os.path.join` instead of hardcoding `/`. We want the script to also work on Windows. Even if before we only used `/`, when writing new code let's try to do the right thing

Reply:
Ok :)

Reply:
@mihaimaruseac As pathlib was added to fix problems like that in os lib so I just used pathlib I hope its ok 📦